docs/source/configuration.rst

```diff
@@ -0,0 +1,36 @@
+Configuration
+=============
+
+
+Bookstore ettings in Jupyter configuration file
+-----------------------------------------------
+
+``~/.jupyter/jupyter_notebook_config.py`` can store commonly used settings for
```

Comment:
I wonder if we should be adopting setting this in `jupyter_config.py` to target the more generic config setup. What do you think @mpacer?

Reply:
Seems reasonable. For context, the name of the config files will match for the name of the JupyterApp you are running. So if you use jupyter_notebook_config.py, it won’t be activated if you run `jupyter nteract`. 

It should be ok since it will only be effectively active on NotebookApp anyway, regardless of the JupyterApp you are running (since it’s a server extension).

I really wish there were a way to tell a JupyterApp to also inherit another JupyterApp’s config file, but that would require a new API in jupyter_core. Using the general config is probably enough for now.